portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +78,20 @@ fetch('/data').then(response => response.json()).then((myData) => {
     const arrayContainer = document.getElementById('array-container');
     arrayContainer.innerText = myData; });
 }
+
+/*
+ * Fetches the comments array and adds each element to the history 
+ * unordered list on the homepage 
+ */
+ function getComments(){
+     fetch('/data').then(response => response.json()).then((comment) => {
```

Comment:
this is a nitpick, but the "comment" variable actually represents an array of comments, right? This variable name is a little bit misleading because it implies a single comment.